src/bucket/BucketManagerImpl.cpp

```diff
@@ -1159,12 +1159,9 @@ loadEntriesFromBucket(std::shared_ptr<Bucket> b, std::string const& name,
               b->getSize(), name, ms, formatSize(bytesPerSec));
 }
 
-std::map<LedgerKey, LedgerEntry>
-BucketManagerImpl::loadCompleteLedgerState(HistoryArchiveState const& has)
+static std::vector<std::pair<Hash, std::string>>
+getBucketHashes(HistoryArchiveState const& has)
```

Comment:
We actually can't use that here, since `HistoryArchiveState::allBuckets` includes future bucket hashes and returns the bucket hashes sorted lexicographically by hash instead of in BucketList order. This was helpful though, I realized we were iterating through the BucketList in reverse order instead of in order. I've removed the `getBucketHashes` function since `loadCompleteLedgerState` and `dumpStateArchivalStatistics` iterate the BubkcetList in different directions.